### ISSUE TITLE
golangci-lint fails during git commit

### ISSUE THREAD

**Issue by floreks** [kubernetes/dashboard#3918]
Whenever I am trying to do a `git commit` and it contains changed go files, `golangci-lint` fails and stops the commit.

```bash
13:31 $ git commit -m "Fix request logging and login view"
husky > pre-commit (node v11.6.0)
  ↓ Stashing changes... [skipped]
    → No partially staged files found...
  ❯ Running linters...
    ✔ Running tasks for src/**/*.ts
    ↓ Running tasks for src/**/*.scss [skipped]
      → No staged files match src/**/*.scss
    ❯ Running tasks for src/**/*.go
      ✖ golangci-lint run -c .golangci.yml --fix src/app/backend/...
        git add



✖ golangci-lint run -c .golangci.yml --fix src/app/backend/... found some errors. Please fix them and try committing again.

level=error msg="Running error: context loading failed: unknown import path \"github.com/kubernetes/dashboard/src/app/backend/handler/filter.go\": cannot find package: failed to analyze"
husky > pre-commit hook failed (add --no-verify to bypass)
```

**Comment by maciaszczykm** [comment]
Happens to me too. @shu-mutou could you take a look on it?

**Comment by shu-mutou** [comment]
@maciaszczykm sure!! 👍

**Comment by shu-mutou** [comment]
As of now, we don't seems to need to add `src/app/backend/...` at the end of `golangci-lint` command in package.json what we added in #3678 to fix #3656.
